### DISCUSSION TITLE
Export to CSV

### DISCUSSION THREAD

**Discussion by rstelma** [code4recovery/12-step-meeting-list#861] [Q&A] [answered]
Dear All,

Thank you for your reply.

Searched through discussions but was not able to find out how to export to CSV.  Where is the menu option for that?

Sorry, I know this question must have been asked a 100 times.

Very much appreciated,
Richard

Version 3.14.5
WordPress 5.9.3

**Comment by tim-rohrer** [comment, marked as answer]
Hi @rstelma,

I transferred this question to the TSML Discussion area. Spec should be reserved for discussions related to the to meeting information exchanges specification.

From your Dashboard, navigate to Meetings->Import & Settings. Under the Import tab, you’ll see an panel on the lower right labeled “Export Meeting List”.

**Comment by rstelma** [comment]
Hi,

Thank you.

I am so sorry. I'm such a dunce. Yes, I see it right there out in the open.
Ding dong. And posting to the wrong group. I'll get better.

Thanks a million,
Richard
